### ISSUE TITLE
Add option to only show filename instead of full path

### ISSUE THREAD

**Issue by renstrom** [tsukkee/unite-tag#29]
When using `:Unite tag`, the path/filename part is basically useless because most of it is truncated and the rest of the space os occupied by the users home folder. Basically this is what all my matches look like:

    @/Users/..spfront.py

That `/Users/` part is useless information, I know what folder/project I'm in. So I would like an option to have the same behaviour as when using `tag/include` which uses `fnamemodify(path, ':t')` instead of `filename` [here](https://github.com/tsukkee/unite-tag/blob/master/autoload/unite/sources/tag.vim#L410-L415).

Perhaps it would be good to have two options, one for the relative path (`fnamemodify(path, ':.')`) and one for just the filename (`fnamemodify(path, ':t')`).

Basically something like this:

Current behaviour:

![screen shot 2015-05-06 at 10 29 53](https://cloud.githubusercontent.com/assets/177685/7489825/31a25828-f3db-11e4-9791-ae634bf67029.png)

`g:unite_source_tag_show_fname_only = 1`

![screen shot 2015-05-06 at 10 30 44](https://cloud.githubusercontent.com/assets/177685/7489848/536d7b54-f3db-11e4-8c5e-8d8d8accf200.png)

`g:unite_source_tag_show_relative_fname = 1`

![screen shot 2015-05-06 at 10 31 27](https://cloud.githubusercontent.com/assets/177685/7489882/9ee4592c-f3db-11e4-8412-a3711f5c010d.png)

**Comment by Shougo** [comment]
Thanks.  I implemented the feature.

**Comment by renstrom** [comment]
Works great, thanks!
